### ISSUE TITLE
BUG: Missing `stats` option fields that should be default enabled.

### ISSUE THREAD

**Issue by elsassph** [FormidableLabs/webpack-stats-plugin#44]
We've been using `webpack-stats-plugin` v0.2.0 with this configuration:
```javascript
new StatsPlugin({
	fields: ['assetsByChunkName', 'assets', 'hash', 'publicPath', 'namedChunkGroups']
}),
```
And the v0.3.0 seems to cause a regression: the `stats.json` file only includes `assetsByChunkName` and the `assets` information.

**Comment by ryan-roemer** [comment]
Hi @elsassph ! Would you be able to get a minimal repository that I can install and run webpack builds against that exhibits the behavior in `0.3.0` but not `0.2.0`? Thanks!

(If you don't have time, I'll see if I have enough time to cobble together something to approximate what I think you have...)

**Comment by ryan-roemer** [comment]
**UPDATE**: Bug is verified. There's something about adding in the `stats` options that messes with defaults.

@elsassph -- While I figure things out more, this should work for you by explicitly enabling stats fields that have been set to false (and I'm not sure why yet):

```js
    new StatsWriterPlugin({
      stats: { publicPath: true, chunkGroups: true },
      fields: ['assetsByChunkName', 'assets', 'hash', 'publicPath', 'namedChunkGroups']
    }),
```

**Comment by elsassph** [comment]
@ryan-roemer I can confirm the `stats` object fixes the issue.

**Comment by ryan-roemer** [comment]
@elsassph -- just released `webpack-stats-plugin@0.3.1` should also fix the issue without needing the extra `stats` configuration field.

**Comment by elsassph** [comment]
@ryan-roemer I can confirm `0.3.1` just works 👍
